utils/api.ts

```diff
@@ -66,6 +66,16 @@ async function postRating(variables: EntityReview.AddVariables): Promise<boolean
   return !!res?.data
 }
 
+async function uploadImage(file: File): Promise<string> {
+  const formData = new FormData()
+  formData.append('file', file)
+  const res = await fetch(`${getImageBaseUrl()}/upload`, {
+    method: 'POST',
+    body: formData
+  }).catch((err) => { console.error(err); return null })
+  return res!.text()
```

Comment:
why ! here? res is not guaranteed to be non-null as you are literally returning null in case of an error. you need to propagate the error upwards to show in the ui. e.g. return null if res is falsy

Reply:
should be fixedish